nest-tax-backend/src/tasks/tasks.service.ts

```diff
@@ -72,20 +72,12 @@ export class TasksService {
           error,
         )
       }
-      if (error instanceof Error) {
-        throw this.throwerErrorGuard.InternalServerErrorException(
-          ErrorsEnum.INTERNAL_SERVER_ERROR,
-          error.message,
-          undefined,
-          undefined,
-          error,
-        )
-      }
       throw this.throwerErrorGuard.InternalServerErrorException(
         ErrorsEnum.INTERNAL_SERVER_ERROR,
-        'Unknown error',
+        error.message,
```

Comment:
this might not exist if it is not instance of Error, should be `error.?message ?? 'Unknown error'`

Reply:
Generic error is also fine. As already implemented.